### ISSUE TITLE
cloc limitation / counted as comment

### ISSUE THREAD

**Issue by lmapii** [AlDanial/cloc#263]
hello,

i've read about the cloc limitations - which are fine for me. i'm not expecting a 100% accurate language-based parsing and am creating this issue just FYI.

i'm using a lot of C-block comments for test-data, very large arrays for which i want to be able to see and jump to the index immediately while debugging without having to check line-numbers:

```
t_test_feed test_cut_stream_in [test_cut_FEED_CNT_IN] =
{

    /*    0 */   {    2u, { 0x01, 0x01,  /* -fill- */ 0x00, 0x00, 0x0
    /*    1 */ , {   18u, { 0xE0, 0x57, 0xB8, 0x1B, 0x1B, 0xD9, 0xCF,
    /*    2 */ , {   22u, { 0xE6, 0x1B, 0x01, 0x04, 0x01, 0x01, 0x76,
    /*    3 */ , {   26u, { 0x70, 0x24, 0x93, 0x1B, 0x01, 0x04, 0x01,
    /*    4 */ , {   22u, { 0x83, 0x1B, 0x1B, 0x04, 0x01, 0x01, 0x5D,
    /*    5 */ , {   18u, { 0xDF, 0x1B, 0x04, 0x0A, 0x01, 0x01, 0x78,
    /*    6 */ , {   29u, { 0x02, 0x93, 0x12, 0x15, 0xFD, 0x04, 0x01,
    /*    7 */ , {    8u, { 0xD8, 0x1B, 0x04, 0xC1, 0x34, 0x94, 0x04,

 ....

    /* 4439 */ , {    6u, { 0x0B, 0xD6, 0x1A, 0x67, 0x2C,
    /* 4440 */ , {   34u, { 0x1B, 0x54, 0x04, 0x01, 0x01,
    /* 4441 */ , {   28u, { 0x1B, 0x01, 0x04, 0x01, 0x01,
    /* 4442 */ , {   32u, { 0x01, 0x04, 0x01, 0x01, 0xD1,
    /* 4443 */ , {   26u, { 0x5A, 0x04, 0x01, 0x01, 0xC7,
    /* 4444 */ , {    3u, { 0x92, 0xF7, 0x04,  /* -fill- 
};
```

these lines are all counted as comments.

**Comment by AlDanial** [comment]
I cut/pasted your code sample to a file (and added a closing ``*/`` after the last ``-fill-``) and cloc had no problem counting the lines correctly.  I also ran with ``--strip-comments`` and it produced the expected output:
<pre>
t_test_feed test_cut_stream_in [test_cut_FEED_CNT_IN] =
{
       {    2u, { 0x01, 0x01,   0x00, 0x00, 0x0
     , {   18u, { 0xE0, 0x57, 0xB8, 0x1B, 0x1B, 0xD9, 0xCF,
     , {   22u, { 0xE6, 0x1B, 0x01, 0x04, 0x01, 0x01, 0x76,
     , {   26u, { 0x70, 0x24, 0x93, 0x1B, 0x01, 0x04, 0x01,
     , {   22u, { 0x83, 0x1B, 0x1B, 0x04, 0x01, 0x01, 0x5D,
     , {   18u, { 0xDF, 0x1B, 0x04, 0x0A, 0x01, 0x01, 0x78,
     , {   29u, { 0x02, 0x93, 0x12, 0x15, 0xFD, 0x04, 0x01,
     , {    8u, { 0xD8, 0x1B, 0x04, 0xC1, 0x34, 0x94, 0x04,
 ....
     , {    6u, { 0x0B, 0xD6, 0x1A, 0x67, 0x2C,
     , {   34u, { 0x1B, 0x54, 0x04, 0x01, 0x01,
     , {   28u, { 0x1B, 0x01, 0x04, 0x01, 0x01,
     , {   32u, { 0x01, 0x04, 0x01, 0x01, 0xD1,
     , {   26u, { 0x5A, 0x04, 0x01, 0x01, 0xC7
     , {    3u, { 0x92, 0xF7, 0x04,
};
</pre>

Might need to supply more code to demonstrate the problem you see.

**Comment by lmapii** [comment]
added complete code of a module as text-file. my output for cloc is this one:

```
---------------------------------------------------------------------------------------------------------
File                                                                  blank        comment           code
---------------------------------------------------------------------------------------------------------
../../dev/test/generated/test_fperm_stream.h                             11             23              9
../../dev/test/generated/test_fperm_stream.c                              6             59              7
```
[test_fperm_stream.txt](https://github.com/AlDanial/cloc/files/1664170/test_fperm_stream.txt)

**Comment by AlDanial** [comment]
My results are quite different:
<pre>
> mv test_fperm_stream.txt test_fperm_stream.c
> cloc test_fperm_stream.c
       1 text file.
       1 unique file.                              
       0 files ignored.

github.com/AlDanial/cloc v 1.75  T=0.01 s (165.0 files/s, 11881.4 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
C                                1              6             13             53
-------------------------------------------------------------------------------
</pre>
Which version of cloc are you using?  Try with the development version,
https://raw.githubusercontent.com/AlDanial/cloc/master/cloc

**Comment by lmapii** [comment]
sorry for the late reply - i'm using 1.74 (installed with homebrew), so seems to be quite outdated
the dev version gives me a compilation error, i guess i don't have all the dependencies installed ?
```
syntax error at cloc line 1, near "400:"
Execution of cloc aborted due to compilation errors.
```

**Comment by AlDanial** [comment]
That error completely baffles me.  Never seen anything like it. 

I released v1.76 since I last posted so your best bet is to try with that.  https://github.com/AlDanial/cloc/releases

**Comment by lmapii** [comment]
yep seems to do the trick, did the upgrade, lines are now as expected. thanks 👍
